attune/io/Topas4.py

```diff
@@ -0,0 +1,79 @@
+from .. import Instrument
+from .. import Tune
+from .. import Arrangement
+from typing import Dict, Union
+import json
+import numpy as np
+
+
+def fromTopas4(Topasfile):
+    # converts a LightConversion marked up Topas4 calibration JSON file into
+    # an Instrument object
```

Comment:
```suggestion
    """Convert a LightConversion marked up Topas4 calibration JSON file into an Instrument object."""
```

Active tense, make into a docstring, which can be inspected from within python